### PR TITLE
Translate more strings into Kurdish 

### PR DESCRIPTION
**Pull request by CYAXXX**

### CODE REVIEW

**Comment by roj1512** [comment]
@GioSensation Please approve the workflow so that we can fix the format errors, if there are any.

**Comment by CYAXXX** [comment]
> Seems like there's some problem with the CI setup. I'm going to ask my colleagues to take a look. In the meantime I've run the tests locally and there are indeed errors. I will let you know in the next few days.

I'll fix it, there were some mistakes done by me, l

**Comment by CYAXXX** [comment]
@roj1512 yek pirsgirêk maye lê nizanim çawa tê çareserkirin, jkx alî min bike bila biqede
2022-11-01 15 12 37 app travis-ci com 4b119d951efd

**Comment by CYAXXX** [comment]
@GioSensation hello, only one issue remains and actually i don't how to fix it, if you can take a look, please.

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -1574,44 +1579,44 @@ msgstr ""
...
 
 msgctxt "SERP footer content"
 msgid "Don't Track Us"
-msgstr ""
+msgstr "Me Neşopîne"
 
 msgctxt "safe search"
 msgid "Don't filter adult content"
-msgstr ""
+msgstr "Naveroka mezinan parzûn neke""
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1590]:
```suggestion
msgstr "Naveroka mezinan parzûn neke"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -1817,43 +1834,45 @@ msgid ""
...
 msgctxt "translations_module"
 msgid "Enter text"
-msgstr ""
+msgstr "Nivîsê têxe"
 
 #. Alias=pseudônimo: pt_pt
 msgid ""
 "Enter the following details: %sName%s: DuckDuckGo%s URL%s: %s Alias%s: d%s"
-msgstr ""
+msgstr "Hûrgiliyên %sNav%s: DuckDuckGo%s URL%s: %s Nasnav%s: d%s" têxe "
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1857]: @CYAXXX There's a `"` sign within the string. Can you please remove it? I think the correct version is:
```suggestion
msgstr "Hûrgiliyên %sNav%s: DuckDuckGo%s URL%s: %s Nasnav%s: d%s"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 3]
```diff
@@ -1989,30 +2010,30 @@ msgstr "Mezinahiya nivîsê"
...
 msgctxt "image-licence"
 msgid "Free to Share and Use"
-msgstr "Azad e bo parvekirin û bikaranîn"
+msgstr "Ji bo parvekirin û bikaranînê Belaş e"
 
 #. a filter for images with the CC BY-ND usage license
 msgctxt "image-licence"
 msgid "Free to Share and Use Commercially"
-msgstr ""
+msgstr ""Ji bo Parvekirin û Bikaranîna Bazirganî belaş e"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2036]:
```suggestion
msgstr "Ji bo Parvekirin û Bikaranîna Bazirganî belaş e"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 4]
```diff
@@ -2363,12 +2385,14 @@ msgstr "Hindî"
 #. Indicate that we have no chart data for prices of the current stock
 msgctxt "Stocks module"
 msgid "Historical prices not available"
-msgstr ""
+msgstr "Bihayên dîrokî ne berdest in""
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:2388]:
```suggestion
msgstr "Bihayên dîrokî ne berdest in"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 5]
```diff
@@ -3182,15 +3206,15 @@ msgstr ""
 
 msgctxt "settingsvalue"
 msgid "Metric (kilograms, meters, Celsius)"
-msgstr ""
+msgstr "Metrî (kilogram, metre, Kelsiyos)""
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:3209]:
```suggestion
msgstr "Metrî (kilogram, metre, Kelsiyos)"
```

**Comment by CYAXXX** [comment]
seems like i'm doing alot of mistakes while translating 😥

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 6]
```diff
@@ -1345,11 +1350,11 @@ msgstr "Wergerê jê bigire"
 
 msgctxt "Report image modal"
 msgid "Copyright violation"
-msgstr ""
+msgstr "Binpêkirina mafê jêgirtinê""
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1353]:
```suggestion
msgstr "Binpêkirina mafê jêgirtinê"
```

**Comment by GioSensation** [comment]
> seems like i'm doing alot of mistakes while translating 😥

No worries 🙂. As I mentioned previously, you can use a software like https://poedit.net/ to help with editing these files locally. It lets you avoid all these little mistakes so you can focus on the translations. It's free for basic usage and works on all desktop platforms. Let me know if you need more info.

Anyway, I've fixed most things. If you can please check [this comment](https://github.com/duckduckgo/duckduckgo-locales/pull/571#discussion_r1011460607) and let me know if that's the correct version we can go ahead. Feel free to hit the "Commit suggestion" button.

**Comment by CYAXXX** [comment]
> > seems like i'm doing alot of mistakes while translating 😥
> 
> No worries 🙂. As I mentioned previously, you can use a software like https://poedit.net/ to help with editing these files locally. It lets you avoid all these little mistakes so you can focus on the translations. It's free for basic usage and works on all desktop platforms. Let me know if you need more info.
> 
> Anyway, I've fixed most things. If you can please check [this comment](https://github.com/duckduckgo/duckduckgo-locales/pull/571#discussion_r1011460607) and let me know if that's the correct version we can go ahead. Feel free to hit the "Commit suggestion" button.

The current provided translated strings will cover most wanted fields, thank you for you time, i was using vs code and online editing, i did not know about poedit that avoid mistakes, sorry for the troubles, i'll use it at next updates.

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 7]
```diff
@@ -1391,21 +1396,21 @@ msgstr ""
...
 #. A setting that *you* choose and is not a default.
 msgctxt "settingsvalue"
 msgid "Custom"
-msgstr ""
+msgstr "Kesane"
 
 msgctxt "settings"
 msgid "Custom %s"
-msgstr ""
+msgstr "Kesane%s"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1409]: Shouldn't we have a space before the placeholder here?

```suggestion
msgstr "Kesane%s"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 8]
```diff
@@ -1391,21 +1396,21 @@ msgstr ""
...
 
 msgctxt "settingsvalue"
 msgid "Custom %s"
-msgstr ""
+msgstr "Kesane%s"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1413]: Shouldn't we have a space before the placeholder here?

```suggestion
msgstr "Kesane %s"
```

Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:1413]: Yeah my bad, should be there a space

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 9]
```diff
@@ -5446,28 +5470,28 @@ msgstr ""
...
 msgid "Title Underline"
-msgstr ""
+msgstr "Binxêzkirina Sernavê"
 
 msgctxt "directions"
-msgid ""
+msgid "Rêgeh"
 "To print directions:%sGo back to the map.%sSelect the route you want to "
 "print.%sClick the print icon.%s"
 msgstr ""
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:5497]: Watch out, this is wrong. You translated the `msgctxt` which is not user-visible. The actual text to translate is `To print directions...`. Please revert this. Thank you 🙂 .

Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:5497]: You could also create a POEditor.com project for this.

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 10]
```diff
@@ -6106,34 +6137,34 @@ msgid "Who We Are"
...
 #. http://i.imgur.com/SOkVdkM.png
 msgctxt "width"
 msgid "Wide"
-msgstr ""
+msgstr "Ferehî"
 
 #. https://duckduckgo.com/params under "Look & Feel Settings"
 msgid "Width:"
-msgstr ""
+msgstr "Ferehî"
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6158]: Mhm, this should be `Width:`, but it looks the same as Wide. Can you please double check and add the `:`, please?

**Comment by roj1512** [comment]
@CYAXXX bibore dema min nebûye heta niha, lê sibê ez ê hewl bidim.

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 11]
```diff
@@ -6106,34 +6137,34 @@ msgid "Who We Are"
...
 msgctxt "feedback form"
 msgid "Wikipedia info"
-msgstr ""
+msgstr "Zanyariyên Wikipedia"
 
 #. Element title explaining this is the win loss record for team, e.g. 10-5
 msgctxt "Sports module"
 msgid "Win/Loss Record"
-msgstr ""
+msgstr "Encama "
```
Comment (GioSensation) [inline locales/ku/LC_MESSAGES/duckduckgo.po:6167]: This looks incomplete. Do you mind double checking?

**Comment by CYAXXX** [comment]
> @CYAXXX bibore dema min nebûye heta niha, lê sibê ez ê hewl bidim.
Tiştek nabe, tenê min mêrik aciz kir ji ber nezaniyê xwe ev tiştek min xemgîn dike :( 
careke dibn ez Poedit bi kar bînim da ku şaşetiyan nîşan bide û ez nekevim rewşeke wisa.

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 11 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po]: done

**Comment by CYAXXX** [comment]
@GioSensation everything is right for now?

**Comment by GioSensation** [comment]
> @GioSensation everything is right for now?

Can you please check https://github.com/duckduckgo/duckduckgo-locales/pull/571/files/4ebe4f6a10f6e8954d1357125071850f17502582#r1011561941 ?

**Comment by roj1512** [comment]
@GioSensation fixed.

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 12]
```diff
@@ -381,7 +381,7 @@ msgstr "Hemû"
 #. a filter for any image with a Creative Commons usage license
 msgctxt "image-licence"
 msgid "All Creative Commons"
-msgstr "Hemû Afirînerê Giştî"
+msgstr "Hemû Afirînerên Giştî"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:384]:
```suggestion
msgstr "Hemû Creative Commons"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 13]
```diff
@@ -1361,23 +1366,23 @@ msgstr ""
 
 #. Instant Answer Tab Name
 msgid "Coupons"
-msgstr ""
+msgstr "Kupon"
 
 msgctxt "video-license"
 msgid "Creative Commons"
-msgstr ""
+msgstr "Afirînerên Giştî"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1373]:
```suggestion
msgstr "Creative Commons"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 14]
```diff
@@ -1431,7 +1436,7 @@ msgstr "Danmarkî"
 #. The name of the Dari language
 msgctxt "language_name"
 msgid "Dari"
-msgstr ""
+msgstr "Darî"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1439]: https://ku.wikipedia.org/wiki/Der%C3%AE_(devok)

```suggestion
msgstr "Derî"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 15]
```diff
@@ -1529,37 +1534,37 @@ msgstr "Rêgeh"
...
+msgstr "Kurteriyan vekole bo ku biçî bo encamên lêgerînê di malperên din de"
 
 #. Button that closes the new messagebox suggesting to add DDG, in the homepage.
 msgid "Dismiss"
-msgstr ""
+msgstr "Paş guh bike"
 
 msgid "Dismiss forever"
-msgstr ""
+msgstr "Heya heyayê paş guh bike"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1552]:
```suggestion
msgstr "Heya heyayê paşguh bike"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 16]
```diff
@@ -1529,37 +1534,37 @@ msgstr "Rêgeh"
...
 
 msgctxt "SERP footer content"
 msgid "Discover shortcuts to go to search results on other sites."
-msgstr ""
+msgstr "Kurteriyan vekole bo ku biçî bo encamên lêgerînê di malperên din de"
 
 #. Button that closes the new messagebox suggesting to add DDG, in the homepage.
 msgid "Dismiss"
-msgstr ""
+msgstr "Paş guh bike"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:1549]:
```suggestion
msgstr "Paşguh bike"
```

**Comment by CYAXXX** [comment]
> > @GioSensation everything is right for now?
> 
> Can you please check https://github.com/duckduckgo/duckduckgo-locales/pull/571/files/4ebe4f6a10f6e8954d1357125071850f17502582#r101156 @GioSensation done by roj

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 17]
```diff
@@ -3423,7 +3423,7 @@ msgstr "Nûçe"
 
 #. http://i.imgur.com/1QgHl7k.png
 msgid "News Articles"
-msgstr "Gotarên nûçryê"
+msgstr "Gotarên nûçeyan"
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:3426]:
```suggestion
msgstr "Gotarên Nûçeyan"
```

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 18]
```diff
@@ -3432,7 +3432,7 @@ msgstr "Na"
 
 msgctxt "noresults"
 msgid "No %s found for %s%s%s."
-msgstr "Tiştek nehate dîtin %s bo %s%s%s."
+msgstr "Tiştek nehate dîtin %sji bo %s%s%s."
```
Comment (roj1512) [inline locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Ti %sek nehate dîtin ji bo %s%s%s?

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 17 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po]: Ya rast bêjim çima tine lê,  mînak info: zanyarî, gelê: alîkarî, ... 
Pêşniyar bi tenê nzm xuya nabe ku rast e lê dîsa em dikarin ser vê mijarê biaxivîn, min nexwest tiştekî nû bikim bêyî poedit lê mixabin şaşitiyek hebû di formekê de ez neçar bûm edit bikim, û min dît ku hinek nehatine wergerandin min wergerandin, ez ê wekî patchek mezin bi riya poedit belav bikim cara tê lê kengê ez jî nizanim

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 18 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Ev format şaş bû, bnr van herdû yan belkî tu alî min bikî 
Screenshot_2022-11-03-19-35-18-121_com vivaldi browser
Screenshot_2022-11-03-19-35-10-210_com vivaldi browser

File: locales/ku/LC_MESSAGES/duckduckgo.po [hunk 19]
```diff
@@ -5172,7 +5172,7 @@ msgstr ""
 
 msgctxt "noresults"
 msgid "Suggestions:"
-msgstr ""
+msgstr "Pêşniyarî:"
```
[the remark this answers is not in the document]
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:5175]: Ya rast bêjim çima tine lê, mînak info: zanyarî, gelê: alîkarî, ...
Pêşniyar bi tenê nzm xuya nabe ku rast e lê dîsa em dikarin ser vê mijarê biaxivîn, min nexwest tiştekî nû bikim bêyî poedit lê mixabin şaşitiyek hebû di formekê de ez neçar bûm edit bikim, û min dît ku hinek nehatine wergerandin min wergerandin, ez ê wekî patchek mezin bi riya poedit belav bikim cara tê lê kengê ez jî nizanim

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 18 above)
Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Heke şaş nebim divê di stringa wergerkirî de wekî stringa sereke çar %s hebin,

Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Tiştek ji gotarên nûçeyê bo "query" nehate dîtin

Wisa çawa ye?

Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Em nikarin zindî pêşdîtin bikin ber wê bi rastî ez jê nizanim... Ka tu serê xwe pê bişîne û nerîna xwe ya dawî pêşniyar bike bo herduyan

Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: > Tiştek ji gotarên nûçeyê bo "query" nehate dîtin
> 
> Wisa çawa ye?

Better? I've no solution to provide honestly

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 19 above)
[the remark this answers is not in the document]
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:5175]: @roj1512  bo mijara suggestions tê got çi?

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 18 above)
Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: bi dilê xwe bike

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 19 above)
[the remark this answers is not in the document]
Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:5175]: nizanim min bi xwe tenê pêşniyar bêyî 'î' dîtiye lê dibe şaş jî bim

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 18 above)
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: Pêşniyara xwe pêşniyar bike an update bike

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 19 above)
[the remark this answers is not in the document]
Comment (CYAXXX) [reply locales/ku/LC_MESSAGES/duckduckgo.po:5175]: Tiştek nabe, hez dikî î jî tevlî bike np4me

File: locales/ku/LC_MESSAGES/duckduckgo.po (hunk 18 above)
Comment (roj1512) [reply locales/ku/LC_MESSAGES/duckduckgo.po:3435]: "Tiştek ji %s bi %s%s%s nehate dîtin."

Sererast ke

**Comment by GioSensation** [comment]
Great work, folks. Much appreciated. Merging now. 🚀

**Pull request merged**